src/component/tooltip/TooltipView.ts

```diff
@@ -472,7 +471,21 @@ class TooltipView extends ComponentView {
 
             let seriesDispatcher: Element;
             let cmptDispatcher: Element;
-            findEventDispatcher(el, (target) => {
+            findEventDispatcher(el, function(target) {
+                // Check if el has any ancestor that has tooltipDisabled: true.
+                let tooltipDisabled = false;
+                let parent = el;
+                while (parent) {
+                    if (parent.tooltipDisabled) {
+                        tooltipDisabled = true;
+                        break;
+                    }
+                    parent = parent.parent;
```

Comment:
The `findEventDispatcher(` itself travels from the original input `el`(include) alone its ancestors.
Thus it's not appropriate to travel to ancestor in the callback.